src/app/view/components/dot-persona-selector/dot-persona-selector.component.ts

```diff
@@ -44,16 +49,40 @@ export class DotPersonaSelectorComponent implements OnInit {
     totalRecords: number;
     value: DotPersona;
     addAction: (item: DotPersona) => void;
+    newPersonaForm: FormGroup;
```

Comment:
Nothing of this form should be in this component, all this issue is about create it owns component. We need to create compoents accordingly, we need to think about this before start coding stuff, we can discuss this with the whole team if we have doubts.